evaluate/task/testing/task.go

```diff
@@ -0,0 +1,69 @@
+package testing
+
+import (
+	"os"
+	"path/filepath"
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/symflower/eval-dev-quality/evaluate/metrics"
+	metricstesting "github.com/symflower/eval-dev-quality/evaluate/metrics/testing"
+	"github.com/symflower/eval-dev-quality/language"
+	"github.com/symflower/eval-dev-quality/model"
+	evaltask "github.com/symflower/eval-dev-quality/task"
+	"github.com/zimmski/osutil"
+)
+
+type TestCaseTaskWriteTests struct {
```

Comment:
Could be any task, right? Not just "write tests".

Reply:
Yes, but right now we just have "write test", and I don't know what other tasks need. Similar as https://github.com/symflower/eval-dev-quality/pull/197#discussion_r1642621456